LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2024 insomniac_lemon
+Copyright (c) 2024 godot-nim
```

Comment:
I recommend either:
Copyright (c) 2024 insomniac_lemon
Copyright (c) 2024 insomniac_lemon, godot-nim

Reply:
This is your work.

Reply:
> This is your work.

After trying to create a filename in the correct folder, when I was typing LICENSE in the filename it I used the license template option. I failed to notice\* that it was applying the new file to the root. I manually reverted it because there doesn't seem to be a revert commit option, at least not unless it's a pull request.

I did create a license in the correct folder after.

\*=that's also somewhat my fault for naming the fork after my project, so it wasn't obvious when it was `/decbinhex4/` instead of `/decbinhex4/decbinhex4/`

Reply:
Oops, I misread the path.